doc/base/classes.xml

```diff
@@ -2800,8 +2800,10 @@
 	</methods>
 	<members>
 		<member name="dialog_hide_on_ok" type="bool" setter="set_hide_on_ok" getter="get_hide_on_ok" brief="">
+			Whether to hide the dialog when accepted. Defaults to true.
```

Comment:
Style guide (https://github.com/GDquest/godot-3-guides/blob/master/contribution-guides/class-reference-styleguide.md) sets the standard for boolean variables.  This should be:

If [code]true[/code], the dialog will be hidden when accepted.  Default value: [code]true[/code].